modules/ROOT/pages/runtime-fabric-logs.adoc

```diff
@@ -408,7 +408,10 @@ You can send a test message to verify connectivity between Runtime Fabric on VMs
 
 Anypoint Runtime Fabric on VMs / Bare Metal also enables you to forward application and cluster logs to external logging solutions other than the third-party solutions supported using log forwarding output plugins. The log forwarder (`rsyslog` client service) built into Runtime Fabric enables you to send log data to an `rsyslog` server over TCP or UDP for viewing, retention, and receiving alerts in a centralized destination. Logging solutions such as Logstash provide methods to receive log data from `rsyslog` clients.
 
-Anypoint Runtime Fabric on VMs / Bare Metal provides dashboards and alerts for critical metrics when performance or availability is compromised.  An SMTP server is required to receive alerts. To view and configure Anypoint Runtime Fabric dashboards and alerts using Ops Center:
```

Comment:
why are we removing this?

Reply:
@kmendiratta-crm I meant to ask about that removal. If you look at where it is in the docs it doesn't make sense 

https://docs.mulesoft.com/runtime-fabric/1.10/runtime-fabric-logs#forward-logs-to-other-external-services